test/compilable/scopeinfer.d

```diff
@@ -10,3 +10,21 @@ static assert(typeof(foo).mangleof == "FNaNbNiNfPvZi");
 auto bar(void* p) { return p; }
 static assert(typeof(bar).mangleof == "FNaNbNiNfPvZQd");
 
+// https://issues.dlang.org/show_bug.cgi?id=19857
+
+struct Stack()
+{
+@safe:
+    int** data;
+    ref int* top()
+    {
+        return *data;
+    }
+}
+
+alias S = Stack!();
+
+//pragma(msg, S.top.mangleof);
+
+version (Win32)
+static assert(S.top.mangleof == "_D4test__T5StackZQh3topMFNaNbNcNiNfZPi");
```

Comment:
What about other platforms?

Reply:
The complete mangling varies slightly from platform to platform, but the issue under test does not, so one platform will suffice.

Reply:
@wilzbach That line needs to `s/4test/10scopeinfer/`

Reply:
Changed.